### ISSUE TITLE
[Conversation] Unable to access properties of Output

### ISSUE THREAD

**Issue by DavidJFowler** [watson-developer-cloud/dotnet-standard-sdk#161]
If I have a dialog node with an "action", e.g.
```
{
  "output": {
    "item": "@enquiryType",
    "action": "lookup"
  }
}
```
As MessageResponse.Output is not a dynamic property I can't access  the action and item properties via the MessageResponse object returned by the Conversation.Message method. I have to use Rest API to do this.

**Comment by mediumTaj** [comment]
@DavidJFowler We dynamic types for these in the `feature-regenerate` branch. We are still waiting on a few more features to be merged in to create an official release.
https://github.com/watson-developer-cloud/dotnet-standard-sdk/tree/feature-regenerate

**Comment by mediumTaj** [comment]
fixed in release `1.3.0`
